Microsoft.DotNet.Cli.sln

```diff
@@ -83,6 +83,50 @@ Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "update-dependencies", "scri
 EndProject
 Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "Microsoft.DotNet.Cli.Utils.Tests", "test\Microsoft.DotNet.Cli.Utils.Tests\Microsoft.DotNet.Cli.Utils.Tests.xproj", "{09C52F96-EFDD-4448-95EC-6D362DD60BAA}"
 EndProject
+Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "NuGet.Common", "..\..\nuget\NuGet.Client\src\NuGet.Core\NuGet.Common\NuGet.Common.xproj", "{98BEE375-A5A0-4FC2-9B7A-25DB41C8045D}"
```

Comment:
Should these be in the solution?

If so, why 2 directories back vs `..\nuget` ?